src/rtcModule/webrtc.cpp

```diff
@@ -823,13 +817,8 @@ void Call::msgCallTerminate(RtMessage& packet)
     {
         EndpointId endpointId(packet.userid, packet.clientid);
         time_t sessionRetryTime = 0;
-        auto itSessionRetyTime = mSessRetriesTime.find(endpointId);
-        if (itSessionRetyTime != mSessRetriesTime.end())
-        {
-            sessionRetryTime = itSessionRetyTime->second;
-        }
-
-        if (sessionRetryTime && (time(NULL) - sessionRetryTime <= RtcModule::kSessSetupTimeout))
```

Comment:
Does it mean we don't need the timeout for session-setup any longer?

Reply:
The timeout is controlled by mSessRetries (std::map<chatd::EndpointId, megaHandle>). If pair (EndpointId, timer handle) exists, there is a session retry in course. When timeout expires, the handle is removed from map